ui/src/App.tsx

```diff
@@ -22,6 +28,44 @@ const TABS: TabsType[] = ['today', 'yesterday', 'archived'];
 function App() {
   const [tab, setTab] = useState<TabsType>('today');
 
+  function CreateButton(i: number){
+    const [open, setOpen] = useState(false);
+
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+
+    const handleClose = () => {
+      setOpen(false);
+    };
```

Comment:
```suggestion
    const handleOpen = useCallback(() => setOpen(true), []);
    
    const handleClose = useCallback(() => setOpen(false), []);
```

See here:

https://react.dev/reference/react/useCallback

Reply:
Makes sense. Applied this change in the new GridButton.tsx file.